docs/intro/installation.rst

```diff
@@ -269,6 +270,15 @@ Python's package manager <fom-macosx-pip>`.
            /sbin
            /Users/MYUSERNAME/Library/Python/3.7/bin
 
+M1 Mac OSX
+""""""""""
+
+As of April 2022, :term:`git-annex` isn't yet available from the `homebrew <https://brew.sh>`_ package manager for M1 Macs, complicating the installation process.
+Our current recommendations are:
+
+#. Check `formulae.brew.sh/formula/git-annex#default <https://formulae.brew.sh/formula/git-annex#default>`_ if there is bottle installation support provided for Apple Silicon. If there is, rely on the installation instructions for :ref:`mac` instead, and consider `opening an issue <https://github.com/datalad-handbook/book/issues/new>`_ to let us know about this.
+#. Perform a manual installation of git-annex. `This step-by-step instruction <https://github.com/fraimondo/csguide-hands-on#problem-3--git-annex-on-m1-mac>`_ has been successfully used to obtain fully functional git-annex installations (with thanks to Fede Raimondo and Vera Komeyer).
```

Comment:
Should we (either now or in future) have these instructions physically in the book as well, as a subsection of http://handbook.datalad.org/en/latest/intro/installation.html#mac-osx?

Reply:
i.e. not linked, in case Fede/Vera's resource changes

Reply:
My gut reaction is no, because we don't even have the infrastructure to ensure they still work - but not a super strong opinion

Reply:
I didn't think of that. So: temporary until proven to work.

Reply:
I gave it a try.  I think the main point from https://github.com/fraimondo/csguide-hands-on#problem-3--git-annex-on-m1-mac is the

```
/usr/sbin/softwareupdate --install-rosetta --agree-to-license
```
and here is some info on Rosetta :

Rosetta is a dynamic binary translator developed by Apple Inc. for macOS, an application compatibility layer between different instruction set architectures. [Wikipedia](https://en.wikipedia.org/wiki/Rosetta_(software))

After that I used datalad-installer

```shell
python3 src/datalad_installer.py git-annex -m autobuild
```
from its github clone (didn't have pip handy), adjusted PATH to point to that installation (didn't want to mess with /usr/local), installed datalad via conda, and `datalad search haxby` worked out as it must ;)

Reply:
filed issue in datalad-installer to make it turnkey  https://github.com/datalad/datalad-installer/issues/103